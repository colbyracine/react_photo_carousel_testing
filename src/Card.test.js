import { render, fireEvent } from "@testing-library/react";
import Carousel from "./Carousel";
import Card from "./Card";

test("it rendered without creashing", () => {
  render(<Card caption />);
});

test("it matched snapshot", () => {
  const { asFragment } = render(
    <Card caption="caption" src="src" currNum="1" totalNum="3" />
  );
  expect(asFragment()).toMatchSnapshot();
});
