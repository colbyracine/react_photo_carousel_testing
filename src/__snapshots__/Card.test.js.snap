// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it matched snapshot 1`] = `
<DocumentFragment>
  <div
    class="Card"
  >
    <h4
      class="Card-title"
    >
      caption
    </h4>
    <img
      alt="caption"
      class="Card-image"
      src="src"
    />
    <small
      class="Card-small"
    >
      Image 1 of 3.
    </small>
  </div>
</DocumentFragment>
`;
